task6.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:io';
+
+const obstaclesPassToWin = 2;
+const jumpFrames = 2;
+const frameTime = 800;
+
+void main() {
+  print('GO GO GODZILA');
+  print('* * * * * * * * * * *');
+
+  bool isGameOver = false;
+  bool isJumping = false;
+  int jumpFramesLeft = 0;
+  int obstaclesPassed = 0;
+  
+  List<int> obstaclePositions = [4, 10];
+  int gamePosition = 0;
+  
+  while (!isGameOver) {
+    if (isJumping) {

```

Comment:
isJumping and no jumping logic has a lot of common. Move this common logic to separete functions and change arguments to make different behaviour